velo/registration/migrations/0005_application_external_invoice_to_local_invoice.py

```diff
@@ -0,0 +1,69 @@
+import requests
```

Comment:
This should not be concatenated as it holds migrations script. Leave it as it is.